packages/react-data-grid/src/masks/InteractionMasks.tsx

```diff
@@ -600,8 +600,8 @@ export default class InteractionMasks<R> extends React.Component<InteractionMask
     const column = columns[draggedPosition.idx];
     const value = getSelectedCellValue({ selectedPosition: draggedPosition, columns, rowGetter });
     const cellKey = column.key;
-    const fromRow = rowIdx < overRowIdx ? rowIdx : overRowIdx;
-    const toRow = rowIdx > overRowIdx ? rowIdx : overRowIdx;
+    const fromRow = rowIdx;
```

Comment:
I feel that we should check the dragging orientation and then set the value respectively.

@MayhemYDG  wdth?

Reply:
We set the correct `overRowIdx` value in `handleDragEnter`, so we just have to use it correctly, which is what this PR does.
Not sure what you're trying to suggest.

Reply:
I think your comment replied me. I thought we need to get the correct id from correct dragging orientation, but as long as this is set properly, I am good.

Reply:
That comment is a reply to you indeed.